examples/okta_inline_hook/basic.tf

```diff
@@ -0,0 +1,18 @@
+resource okta_inline_hook test {
+  name    = "testAcc_replace_with_uuid"
+  version = "1.0.1"
+  type    = "com.okta.oauth2.tokens.transform"
+
+  channel = {
+    type    = "HTTP"
+    version = "1.0.0"
+    uri     = "https://id-mapper.articulate.com/v1/users"
```

Comment:
Can you obfuscate this in the public example please?

Reply:
9996f7f7d908f119efd9dc949730acd335c64162